src/NexusMods.App.UI/Overlays/Download/Cancel/CancelDownloadOverlayViewModel.cs

```diff
@@ -5,17 +5,17 @@ namespace NexusMods.App.UI.Overlays.Download.Cancel;
 
 public class CancelDownloadOverlayViewModel : AViewModel<ICancelDownloadOverlayViewModel>, ICancelDownloadOverlayViewModel
 {
-    public IDownloadTaskViewModel DownloadTask { get; } = new DownloadTaskDesignViewModel();
+    public IEnumerable<IDownloadTaskViewModel> DownloadTasks { get; } = Enumerable.Empty<IDownloadTaskViewModel>();
```

Comment:
Using `IEnumerable<T>` in VMs isn't ideal, since they don't have `Length`/`Count` property. I'd suggest some form of `IList<T>` like `IReadOnlyList<T>`.